freqtrade/tests/optimize/test_hyperopt.py

```diff
@@ -618,3 +618,77 @@ def test_continue_hyperopt(mocker, default_conf, caplog):
 
     assert unlinkmock.call_count == 0
     assert log_has(f"Continuing on previous hyperopt results.", caplog)
+
+
+def test_print_json_spaces_all(mocker, default_conf, caplog, capsys) -> None:
+    dumper = mocker.patch('freqtrade.optimize.hyperopt.dump', MagicMock())
+    mocker.patch('freqtrade.optimize.hyperopt.load_data', MagicMock())
+    mocker.patch(
```

Comment:
you could combine these 3 options using `mocker.patch.multiple` ... but again, that's for the future :+1:

Reply:
I just copied it from another test above changing the options passed to hyperopt and the asserts for output -- still not very familiar with all these mocking magic...